app/views/caravans/index.html.erb

```diff
@@ -4,7 +4,7 @@
       <% @caravans.each do |caravan| %>
       <div class="col-3">
         <div class="card h-100">
-          <div class="card-image" style="background-image: url(https://images.unsplash.com/photo-1597192535374-aeba30156218?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80)">
+          <div class="card-image" style= "background-image: url(<%= cl_image_path(caravan.photo.key) if caravan.photo.attached? %>)">
```

Comment:
do we want to add a 'fallback' image in case the user doesn't upload a photo?

Reply:
Yeah, great idea.

Reply:
ok